core/src/main/java/com/graphhopper/routing/lm/LandmarkSuggestion.java

```diff
@@ -40,7 +40,7 @@ public BBox getBox() {
      * to specify an explicit bounding box. TODO: support GeoJSON instead.
      */
     public static final LandmarkSuggestion readLandmarks(String file, LocationIndex locationIndex) throws IOException {
-        // landmarks should be suited for all vehicles
+        // landmarks should be suited for all profiles
```

Comment:
Yes, I was just wondering as it's stating "suited" meaning, it's a good idea to combine PT with LM. I am not sure if this is true, so I was asking.